packages/nimble-components/src/patterns/button/styles.ts

```diff
@@ -78,24 +80,57 @@ export const styles = css`
     }
 
     .control:hover {
+        border-color: ${borderHoverColor};
         box-shadow: 0px 0px 0px ${borderWidth} ${borderHoverColor} inset;
-        outline: none;
     }
 
     .control${focusVisible} {
+        border-color: ${borderHoverColor};
         box-shadow: 0px 0px 0px ${borderWidth} ${borderHoverColor} inset;
-        outline: ${borderWidth} solid ${borderHoverColor};
-        outline-offset: -4px;
     }
 
     .control:active {
         box-shadow: none;
         outline: none;
+        background-image: linear-gradient(
+            ${fillSelectedColor},
+            ${fillSelectedColor}
+        );
+        background-size: calc(100% - 2px) calc(100% - 2px);
+        background-repeat: no-repeat;
+        background-position: center;
```

Comment:
These only apply when a background-image is set, so we can just move them up to the `.control {}` selector and avoid duplication (remove all the other instances of them). Whenever an image is set these are always the same